packages/common/test-configs/jest-config-base.ts

```diff
@@ -10,6 +10,7 @@ const config: InitialOptionsTsJest = {
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
   moduleNameMapper: {
     'lodash-es': 'lodash',
+    '\\.(css|scss)$': 'identity-obj-proxy',
```

Comment:
@florkbr From what I can tell this would apply across the board and not just to node_modules.

Reply:
@vidyanambiar alright - I'm not sure if this will have unintended consequences so I'd like to follow up on this in the future. Can you open a ticket? Specifically to either:
1. Scope down this rule somehow
2. Actually enable our Jest config to properly load css/scss